pandas/core/groupby.py

```diff
@@ -929,6 +929,10 @@ def mean(self):
         Compute mean of groups, excluding missing values
 
         For multiple groupings, the result index will be a MultiIndex
+
+        Do not call this function via the numpy library. numpy's mean
```

Comment:
you are just adding unecessary documentation. we are trying to avoid people EVER calling numpy directly. no need to encourage it.